src/vs/workbench/browser/actions/workspaceCommands.ts

```diff
@@ -131,6 +131,11 @@ interface IOpenFolderAPICommandOptions {
 	forceReuseWindow?: boolean;
 	noRecentEntry?: boolean;
 	forceLocalWindow?: boolean;
+	filesToOpen?: {
+		uri: URI;
+		line?: number;
```

Comment:
I wonder whether this should align closer to the API we allow for `vscode.open`:

https://github.com/microsoft/vscode/blob/292cc7f4c48fd3f95d6d3709a0d5872ec35a5039/src/vs/workbench/browser/parts/editor/editorCommands.ts#L492-L492

Where we allow a `ITextEditorSelection`:

https://github.com/microsoft/vscode/blob/22cf36faf81ca270390ec233389a81339b40b63a/src/vs/platform/editor/common/editor.ts#L297-L302

Unfortunately, it doesn't look like we support full selection though from where we pass the URI on:

https://github.com/microsoft/vscode/blob/7ac9328da83e00305eb393da74d110b81a1a6204/src/vs/workbench/contrib/files/browser/fileCommands.ts#L94-L94